lib/ansible/plugins/connection/winrm.py

```diff
@@ -25,18 +25,23 @@
 import shlex
 import traceback
 import json
-import xmltodict
 
-from ansible.compat.six.moves.urllib.parse import urlunsplit
 
+from ansible.compat.six.moves.urllib.parse import urlunsplit
 from ansible.errors import AnsibleError, AnsibleConnectionFailure
+
 try:
     import winrm
     from winrm import Response
     from winrm.protocol import Protocol
 except ImportError:
     raise AnsibleError("winrm is not installed")
 
+try:
+    import xmltodict
+except ImportError:
+    raise AnsibleError("winrm is not installed")
+
```

Comment:
Since we don't actually need those present and functional to build docs, could just set HAS_PYWINRM/HAS_XMLTODICT and check them in CP init instead of raising immediately (though I suspect we may run into similar dependency issues in other plugins)